DP03_05_Near-Earth_Objects.ipynb

```diff
@@ -0,0 +1,1395 @@
+   "execution_count": null,
+   "id": "2d5e15fd-cb86-40e3-8cdc-ca7e49023b29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.style.use('tableau-colorblind10')\n",
+    "%matplotlib inline\n",
+    "\n",
+    "params = {'axes.labelsize': 15,\n",
+    "          'font.size': 15,\n",
+    "          'legend.fontsize': 10,\n",
+    "          'xtick.major.width': 2,\n",
+    "          'xtick.minor.width': 1,\n",
+    "          'xtick.major.size': 6,\n",
+    "          'xtick.minor.size': 3,\n",
+    "          'xtick.direction': 'in',\n",
+    "          'xtick.top': True,\n",
+    "          'lines.linewidth': 2,\n",
+    "          'axes.linewidth': 2,\n",
+    "          'axes.labelweight': 1,\n",
+    "          'axes.titleweight': 1,\n",
+    "          'ytick.major.width': 1,\n",
+    "          'ytick.minor.width': 2,\n",
+    "          'ytick.major.size': 6,\n",
+    "          'ytick.minor.size': 3,\n",
+    "          'ytick.direction': 'in',\n",
+    "          'ytick.right': True,\n",
+    "          'figure.figsize': [6, 6],\n",
+    "          'figure.facecolor': 'White'\n",
+    "          }\n",
+    "plt.rcParams.update(params)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cb24cb9-687f-46b2-aec6-b51fff5be345",
+   "metadata": {},
+   "source": [
+    "## 2. Orbital Parameters of Near-Earth Objects (NEOs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eff43554-b31a-4b1b-95a3-6951f46e6bc9",
+   "metadata": {},
+   "source": [
+    "### 2.1 Query the DP0.3 catalogs to create a table of NEOs and their orbital parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f29090e8-de29-4d0d-949c-2276a8ba4cc8",
+   "metadata": {},
+   "source": [
+    "Get an instance of the Rubin TAP service client, and assert that it exists."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6d7d436c-ffc8-4e67-9662-36467e9c7480",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "service = get_tap_service(\"ssotap\")\n",
+    "assert service is not None"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "620679de-2f93-4227-8d06-2df6cdc49224",
+   "metadata": {},
+   "source": [
+    "Query the DP0.3 catalogs, joining the MPCORB and SSObject tables on their ssObjectId. Limit the query to select sources in the MPCORB table with perihelia $q$ < 1.3 au, eccentricities $e$ < 1, and semimajor axes $a$ < 4 au to get only near-Earth objects, excluding interstellar objects with $e$ > 1 or $a$ < 0.\n",
+    "\n",
+    "Define the table returned by this query as \"uniqueNEOs\" since it contains the IDs of unique solar system objects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8952dd2a-48dc-4ca6-b986-91b8a633acfc",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "q_select = '1.3'\n",
+    "e_select = '1.0'\n",
+    "a_select = '4.0'\n",
+    "\n",
+    "query = \"\"\"\n",
```

Comment:
add line so """ is below "query" to be consistent

Reply:
This query statement was copied from notebook DP03_04a_Introduction_to_Phase_Curves.ipynb for consistency and modified as needed. I can move the """ if needed.